packages/forms-web-app/src/pages/process-guide/_translations/en.json

```diff
@@ -0,0 +1,69 @@
+{
+	"applicationSummary": {
```

Comment:
Would it better to call this key 'index' as it relates to the content on the index page? Also, it might be good to scope the index translations to the index folder and only have the shared process guide content here, will help keep the files from getting too large to maintain. @emil-placheta @cprieststephens Would be good to have a chat about this pattern.